tests/cmake_integration/Messenger/C++11_Messenger/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 project(OpenDDS_C++11_Messenger CXX)
 cmake_minimum_required(VERSION 3.8.2)
 
-find_package(OpenDDS REQUIRED)
+if(NOT OpenDDS_FOUND)
```

Comment:
I wouldn't have thought this should be necessary, but I tried it locally and do see that it fails on multiple calls. I don't think we can fix this now, so it's fine for now. I'd like this to be possible in the future though since I think I've seen code for this is other CMake modules.